RecoPPS/Local/plugins/CTPPSLocalTrackLiteReAligner.cc

```diff
@@ -0,0 +1,107 @@
+/****************************************************************************
+ *
+ * This is a part of TOTEM offline software.
+ * Authors:
+ *   Jan Kašpar (jan.kaspar@gmail.com)
+ *
+ ****************************************************************************/
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/CTPPSReco/interface/CTPPSLocalTrackLite.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSLocalTrackLiteFwd.h"
+
+#include "CondFormats/AlignmentRecord/interface/RPRealAlignmentRecord.h"
+#include "CondFormats/PPSObjects/interface/CTPPSRPAlignmentCorrectionsData.h"
+
+//----------------------------------------------------------------------------------------------------
+
+class CTPPSLocalTrackLiteReAligner : public edm::stream::EDProducer<> {
```

Comment:
it looks like this can be a global module
```suggestion
class CTPPSLocalTrackLiteReAligner : public edm::global::EDProducer<> {
```

Reply:
No problem to make this change. For my education, isn't it better to conceive modules as "stream" since this offers better parallelisation possibilities?

Reply:
Intances of "stream" modules are replicated in every stream, while "global" modules are run only once independently on the number of streams. If you don't need to have different instances per stream (e.g non-const class members that can be modified independently in every stream), "global" modules must be preferred.

Reply:
Thanks for explanation! Changed to global.